### ISSUE TITLE
'Write HTTP clients' Use HttpServer instead of Stream<HttpRequest>

### ISSUE THREAD

**Issue by dark-chocolate** [dart-lang/site-www#3163]
Page URL: https://dart.dev/tutorials/server/httpserver
Page source: https://github.com/dart-lang/site-www/tree/master/src/_tutorials/server/httpserver.md

Description of issue:

Referring the code written [here](https://dart.dev/tutorials/server/httpserver#using-http-server-package):

```dart
Future main() async {
  Stream<HttpRequest> server;
  // ...
}
```

But we can't get the address and port number from the `Stream<HttpRequest>`. We should rather change it to 

```dart
HttpServer server;
```

in order to allow 
```dart
print('http://${server.address.address}:${server.port}');
```

**Comment by kwalrath** [comment]
Also /cc @kevmoo @mit-mit

**Comment by Sameerkash** [comment]
> /cc @Sameerkash, who volunteered to work on #3128.

Yes, me and @kevmoo are in constant communication trying to rewrite this page to use shelf. 

This should be up soon

**Comment by kwalrath** [comment]
Marking this as blocked since I don't want to close it, but it should be addressed by #3128.

**Comment by atsansone** [comment]
This page is now obsolete. It has been changed to a reference to other server resources. #3128 has been closed. Closing.
